pkgs/applications/networking/p2p/stig/default.nix

```diff
@@ -44,7 +44,9 @@ python3.pkgs.buildPythonApplication rec {
   ];
 
   checkPhase = ''
-    pytest tests
+    LC_ALL=en_US.utf8 pytest tests \
+      --deselect=tests/commands_test/misc_cmds_test.py::TestSetCmd::test_setting_with_eval \
```

Comment:
I had all the tests pass without these `--deselect` flags - on NixOS. Did you disable them for Darwin?

Reply:
that is weird, for me `test_setting_with_eval` was failing on NixOS, and `test_string__month_day_hour_minute_second` was only failing on Darwin.

I'm going to see if `test_setting_with_eval` fails on OfBorg and update accordingly

Reply:
I've removed el deselect of`test_setting_with_eval` and added a comment about `test_string__month_day_hour_minute_second`